### PR TITLE
Add tracking for core actions

### LINKED ISSUE #1439: Add analytics tracking for core actions
# Problem

- [ ] Clicked New Task

### PR DESCRIPTION
**Pull request by JoshSmith**
# What's in this PR?

Work in progress. So far:

- [x] Clicked New Task
- [x] Clicked on Task Card in List
- [ ] Clicked Assign Task on Task Card (track organization, project, task)
- [x] Clicked GitHub Link on Task Card (track organization, project, task)
- [x] Clicked Install GitHub Application (track organization and project)
- [x] Clicked Connect with GitHub

## References
Progress on #1439

### CODE REVIEW

**Comment by JoshSmith** [comment]
Does not have tests. Not sure how to approach this yet.

**Comment by JoshSmith** [comment]
`<a href="http://google.com" {{action "logClick" preventDefault=false}}>Google!</a>`

File: app/components/github/install-link.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import ENV from 'code-corps-ember/config/environment';
+import Component from '@ember/component';
+import { get } from '@ember/object';
+import { inject as service } from '@ember/service';
```
Comment (begedin) [inline app/components/github/install-link.js]: I like the new format. Have to get used to using it, though.

File: tests/acceptance/task-list-test.js [hunk 2]
```diff
@@ -78,3 +79,41 @@ test('member can assign/reassign/unassign tasks to user', function(assert) {
...
+        organization: organization.name,
+        organization_id: organization.id,
+        project: project.title,
+        project_id: project.id,
+        task: task.title,
+        task_id: task.id
+      }, 'Event tracking was called with proper attributes.');
+    }
+  }));
+  this.application.inject('route', 'metrics', 'service:stubMetrics');
```
Comment (begedin) [inline tests/acceptance/task-list-test.js]: While I'm positive that the `inject` step is needed, I'm not sure if the `instantiate: false` line is necessary, or if we need to stub with an `Ember.Service`. An `Ember.Object` or even a POJO might be enough.

I don't think this matters too much though, so you can decide if it's worth the time to potentially save an import and a line or two.

Comment (JoshSmith) [reply tests/acceptance/task-list-test.js]: I'll leave it for now. Not a big priority relative to other things.

**Pull request merged**